libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -1070,6 +1077,151 @@ void ofShader::setUniforms(const ofParameterGroup & parameters) const{
 		}
 	}
 }
+
+
+//--------------------------------------------------------------
+template<typename T>
+void ofShader::setDefineConstantTemp(const string & name, T value){
+#if (!defined(TARGET_LINUX) || defined(GCC_HAS_REGEX))
```

Comment:
Makes sense =) I thought I saw some subtle differences in how vs implements std::regex, I can do some testing on windows...